doc/source/advanced-use.rst

```diff
@@ -6,246 +6,256 @@ Clones
 ------
 
 When a jail is cloned, iocage creates a ZFS clone filesystem.
-In a nutshell clones are cheap lightweight writable snapshots.
+Essentially, clones are cheap, lightweight, and writable snapshots.
 
-A clone depends on its source snapshot and filesystem.
-If you'd like to destroy the source jail and preserve its clones
-you need to promote the clone first, otherwise the source jail cannot be destroyed.
+A clone depends on its source snapshot and filesystem. To destroy the
+source jail and preserve its clones, the clone must be promoted first.
 
 Create a clone
 ++++++++++++++
 
-To clone www01 to www02 run:
+To clone **www01** to **www02**, run:
 
-``iocage clone www01 tag=www02``
+:samp:`# iocage clone www01 tag=www02`
 
-To clone a jail from an existing snapshot:
+Clone a jail from an existing snapshot with:
 
-``iocage clone www01@snapshotname tag=www03``
+:samp:`# iocage clone www01@snapshotname tag=www03`
 
 Promoting a clone
 +++++++++++++++++
 
-**To promote a cloned jail, simply run:**
+**To promote a cloned jail, run:**
 
-``iocage promote UUID | TAG``
+:samp:`# iocage promote UUID | TAG`
 
-The above step will reverse the clone and source jail relationship.
-Basically the clone will become the source and the source jail will be demoted to a clone.
+This reverses the *clone* and *source* jail relationship. The clone
+becomes the source and the source jail is demoted to a clone.
 
-**Now you can remove the demoted jail with:**
+**The demoted jail can now be removed:**
 
-``iocage destroy UUID | TAG``
+:samp:`# iocage destroy UUID | TAG`
 
 Updating jails
 --------------
 
-Updates are handled with the freebsd-update(8) utility. Jails can be updated
-while they are stopped or running.
+Updates are handled with the freebsd-update(8) utility. Jails can be
+updated while they are stopped or running.
 
-To update a jail to latest patch level run:
+To update a jail to latest patch level, run:
 
-``iocage update UUID | TAG``
+:samp:`# iocage update UUID | TAG`
 
-This will create a back-out snapshot of the jail automatically.
+This automatically creates a back-out snapshot of the jail.
 
-When finished with updating and the jail is working OK, simply remove the snapshot:
+When updates are finished and the jail appears to function properly,
+remove the snapshot:
 
-``iocage snapremove UUID|TAG@snapshotname``
+:samp:`# iocage snapremove UUID|TAG@snapshotname`
```

Comment:
s/TAG/NAME/ and the user is now responsible for the snapshots, that may change in the future if I port that functionality back, but currently we don't do it automatically.